Cargo.toml

```diff
@@ -11,15 +11,15 @@ keywords = [ "crypto", "bitcoin" ]
 readme = "README.md"
 
 [dependencies]
-bitcoin = { version= "0.21", features=["serde"]}
+bitcoin = { version= "0.23", features=["use-serde"]}
 bitcoin_hashes={version="0.7", features=["serde"]}
-secp256k1 = "0.15"
+secp256k1 = "0.17"
```

Comment:
You can remove the `secp256k1` and `bitcoin_hashes` dependencies.

They are exposed as `bitcoin::secp2156k1` and `bitcoin::hashes`.

Reply:
fixed